Automattic/newspack-ads
# chore: update Prebid.js

[Version 6.12.0 of Prebid.js](https://github.com/prebid/Prebid.js/releases/tag/6.12.0) released https://github.com/prebid/Prebid.js/pull/7969, so we no longer need to maintain our own version of the babel plugin used for building Prebid.js.

This PR removes the pbjsGlobals plugin and restores the use of the native babel config.

### How to test

1. Run `npm ci && npm run build`
2. Confirm the build succeeds 
3. Confirm Prebid.js is working in debug mode (auction logs in console)
